globus-requirements.txt

```diff
@@ -0,0 +1,3 @@
+# extra requirements for globus
+-r ./requirements.txt
+globus_sdk[jwt]>=1.0.0,<2.0.0
```

Comment:
I looked into doing authentication without the globus_sdk here, but most people will want to initiate file transfers using the globus_sdk from within their python notebooks, so they would end up installing it anyway.